backend/Api/Validators/ConsultantValidators.cs

```diff
@@ -0,0 +1,43 @@
+using System.Net.Mail;
+using Api.Routes;
+using Core.DomainModels;
+
+namespace Api.Validators;
+
+public static class ConsultantValidators
+{
+    public static bool IsValidEmail(string email)
+    {
+        var valid = true;
+
+        try
+        {
+            var mailAddress = new MailAddress(email);
+        }
+        catch
+        {
+            valid = false;
+        }
+
+        return valid;
+    }
+
+    public static IDictionary<string, string []> ValidateUniqueness(List<Consultant> consultantList,
+        ConsultantApi.ConsultantWriteModel body)
+    {
+        var results = new Dictionary<string, string[]>();
+
+        var isNameUnique = consultantList.All(item => item.Name != body.Name);
+        var isEmailUnique = consultantList.All(item => item.Email != body.Email);
```

Comment:
```suggestion
        var isNameUnique = consultantList.Any(item => item.Name == body.Name);
        var isEmailUnique = consultantList.Any(item => item.Email == body.Email);
```

As a general readability-tip; positive logic is usually more easily parsed by humans. 
Though this is logically equal, think about how it reads in your mind: 

"All consultants should have a name thats not <name>" 
or
"Any consultant with a name <name>"